src/Sylius/Bundle/ResourceBundle/DependencyInjection/Driver/Doctrine/DoctrinePHPCRDriver.php

```diff
@@ -68,6 +70,32 @@ protected function addRepository(ContainerBuilder $container, MetadataInterface
      */
     protected function addDefaultForm(ContainerBuilder $container, MetadataInterface $metadata)
     {
+        $options = array_merge(
+            [
+                'default_parent_path' => null,
+                'autocreate' => false,
+            ],
+            $metadata->hasParameter('options') ? $metadata->getParameter('options') : []
+        );
+
+        $defaultFormBuilderDefinition = new Definition(DefaultFormBuilder::class);
+        $defaultFormBuilderDefinition->setArguments([
+            new Reference($metadata->getServiceId('manager')),
+            $options['default_parent_path'],
+            $options['autocreate']
```

Comment:
There options only apply to the `DefaultFormBuilder` - which means that as soon as the user creates a custom form type these values become meaningless.

I wonder if these should be implemented as a PHPCR-ODM EventSubscribers instead.